### PR TITLE
feature/tour page main section fetch & render data

### PR DESCRIPTION
**Pull request by Ahmed-Fahmy-Dev**
- tour name rendered as h1
- about tour rendered with content
- reasons to choose rendered with content

### CODE REVIEW

File: client/src/pages/TourPage/TourPage.js [hunk 1]
```diff
@@ -147,19 +147,21 @@ const TourPage = () => {
...
 
   useEffect(() => {
     dispatch(fetchTour());
   }, []);
 
   return (
     <>
+      <div>
+        <ImageGallery />
+      </div>
```
Comment (HIMDanny) [inline client/src/pages/TourPage/TourPage.js]: Suggest deleting it cause `ImageGallery` already placed in `HeaderContent > Container`

File: client/src/pages/TourPage/TourPage.js [hunk 2]
```diff
@@ -147,19 +147,21 @@ const TourPage = () => {
...
       <HeaderContent>
         <Container>
           <Typography align="center" variant="h1" mt={17} mb={5} fontSize="50px">
```
Comment (HIMDanny) [inline client/src/pages/TourPage/TourPage.js]: Suggest capitalizing first letter of the title

```suggestion
          <Typography
            align="center"
            variant="h1"
            mt={17}
            mb={5}
            fontSize="50px"
            sx={{
              ':first-letter': {
                textTransform: 'capitalize',
              },
            }}
          >
```

File: client/src/pages/TourPage/TourPage.js (hunk 1 above)
Comment (Ahmed-Fahmy-Dev) [reply client/src/pages/TourPage/TourPage.js]: done

File: client/src/pages/TourPage/TourPage.js (hunk 2 above)
Comment (Ahmed-Fahmy-Dev) [reply client/src/pages/TourPage/TourPage.js]: done

**Review by AltysS — APPROVED** [review]

**Comment by AltysS** [comment]
Great job. Nothing to add here

**Review by HIMDanny — APPROVED** [review]

**Pull request merged**